client/src/components/Image.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import axios from 'axios'
+import { Link } from 'react-router-dom'
+import { AuthConsumer, } from "../providers/AuthProvider";
+
+class Image extends React.Component {
```

Comment:
Calling this component Image might conflict later on if we use a semantic Image. Maybe instead we can call it Images? Or ImagesContainer? ShowImages? Something like that.